src/courtroom.cpp

```diff
@@ -1306,6 +1323,12 @@ void Courtroom::update_character(int p_cid)
       }
     }
   }
+  if (is_ao2_bg) {
+    set_size_and_pos(ui_vp_chatbox, "ao2_chatbox", f_char);
+  }
+  else {
+    set_size_and_pos(ui_vp_chatbox, "chatbox", f_char);
+  }
```

Comment:
This backward compatibility thing needs to go. We can save this for another branch though.